requirements.txt

```
@@ -1 +1 @@
-autopep8==1.5.4
+autopep8==v2.0.1
```

Not sure if it works with the leading `v`, so best to remove.
```suggestion
autopep8==2.0.1
```